corehq/form_processor/sql_functions/deprecate_form.sql

```diff
@@ -0,0 +1,29 @@
+DROP FUNCTION IF EXISTS deprecate_form(TEXT, TEXT, TIMESTAMP);
+
+CREATE FUNCTION deprecate_form(new_form_id TEXT, orig_id TEXT, edited_on TIMESTAMP) RETURNS VOID AS $$
```

Comment:
@snopoke just an fyi, i was doing some reading on these for prep and you can reference the variable names as the alias you set in the function signature. i.e. you should be able to reference `new_form_id` instead of `$1`. improves some of the readability. http://www.postgresql.org/docs/9.1/static/plpgsql-declarations.html